instrumentation/tomcat/tomcat-common/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/tomcat/common/TomcatServerHandlerInstrumentation.java

```diff
@@ -36,6 +40,14 @@ public void transform(TypeTransformer transformer) {
             .and(named("service"))
             .and(takesArgument(0, named("org.apache.coyote.Request")))
             .and(takesArgument(1, named("org.apache.coyote.Response"))),
-        adviceClassName);
+        handlerAdviceClassName);
+
+    transformer.applyAdviceToMethod(
+        isMethod()
+            .and(named("postParseRequest"))
```

Comment:
`Adapter` doesn't have this method. Is there any reason why we're not instrumenting `CoyoteAdapter`?

Reply:
Thanks. Didn't even realize that it wasn't using `CoyoteAdapter`.